### PR TITLE
Feature/no ticket/multiple kafka server

### PR DESCRIPTION
**Pull request by E-ricus**
## Description
Change dependency to a forked one with upgraded dependencies 
Admit multiple kafka hosts
## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [x] I did add reviewers to this PR
- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I created the unit tests and they are passing

### CODE REVIEW

File: go.mod [hunk 1]
```diff
@@ -4,7 +4,7 @@ go 1.13
 
 require (
 	cloud.google.com/go/pubsub v1.2.0
-	github.com/dangkaka/go-kafka-avro v0.0.0-20191011061810-51334c900b84
+	github.com/ericpubu/go-kafka-avro v0.0.2
```
Comment (daltemen) [inline go.mod:7]: Is this fork not possible to do in the Merlin github organization?

Comment (E-ricus) [reply go.mod:7]: It's possible, but I'll keep upgrading in my fork to make a PR eventually on the main repository, so using that fork would give more functionalities

Comment (E-ricus) [reply go.mod:7]: as soon as the PR is merged I'll return the dependency to the main repo

Comment (daltemen) [reply go.mod:7]: nice

**Review by daltemen — APPROVED** [review]

**Review by oscanezing — APPROVED** [review]

**Pull request merged**